docs/en/writing-running-appium/caps.md

```diff
@@ -165,6 +165,7 @@ These Capabilities are available only on the [Espresso Driver](/docs/en/drivers/
 |Capability|Description|Values|
 |----|-----------|-------|
 |`espressoServerLaunchTimeout`|Timeout in milliseconds used to wait for an espresso server to launch. Defaults to `30000` |e.g., `50000`|
+|`skipServerInstallation`|Skip Espresso server build and apk installation. Can be used to improve startup performance when an Espresso server and the proper the app under test is already installed on the device. Defaults to `false` | `true` or `false`|
```

Comment:
This option could potentially be dangerous if appium version is changed, because it forcefully skips all the checks. This might render Espresso to be completely broken until the rebuilt server is installed.
That is why it only makes sense to use the cap after one makes sure the correct server version is already installed on the device under test and there is no need to run any verifications for that on Appium side.